### ISSUE TITLE
When playing is originally set to false, onProgress does not return played property

### ISSUE THREAD

**Issue by juancarlosfarah** [CookPete/react-player#79]
I'm using the player with controls set to `true` and when I set `playing` to `false`, I do not see the `played` property in the object the `onProgress` function is returning. It looks to me that the following clause is not working as expected because `_this.props.playing` is staying as `false` even after I press play.

``` javascript
if (played !== null && played !== _this.prevPlayed && _this.props.playing) {
    progress.played = _this.prevPlayed = played;
}
```

**Comment by hectoroso** [comment]
Yes, I've just run into this issue as well. When playing=true, the argument passed to the onProgress callback look like:
```
Object {loaded: 0.7045520628048978, played: 0.027777565987141796}
```
When playing=false:
```
Object {loaded: 0.568733240719801}
```

**Comment by CookPete** [comment]
This was a deliberate decision so that `onProgress` only includes values *when they change*. It was to prevent state from being constantly updated even if none of the values are changing (eg when the media is fully loaded but paused I don't want to be re-rendering every 500ms or so).

Perhaps `onProgress` should always output both values just for clarity, and we leave it up to the developer to prevent further updates when the values are the same on every `onProgress`.

> _this.props.playing is staying as false even after I press play.

The `playing` prop is passed in to the player by you, so whether it is true or false is up to you. If you are pressing play on the native player controls, you should be listening to `onPlay` and updating the state of your app to reflect that it has been played by the user. Notice [how the demo app does it](https://github.com/CookPete/react-player/blob/a76daa27eaf9dba7013f9868664fa4fdb9fb3407/src/demo/App.js#L98-L99) so that playing/pausing using the native player controls updates the external controls.

**Comment by juancarlosfarah** [comment]
Thanks for your response, @CookPete. That makes sense. I ended up doing as you said and updating the state with functions listening to `onPlay` and `onPause`. Initially it did throw me off that the object returned by `onProgress` was missing the `played` property, but I'm not sure how best to deal with that. Perhaps just make it clear in the documentation? I don't think that particular bit is clear.

**Comment by CookPete** [comment]
>I don't think that particular bit is clear.

I agree. To be honest it seems cleaner and less confusing to just include both values all the time, so I'll make the changes, update the docs and publish a new version.

**Comment by CookPete** [comment]
Should be fixed in `0.7.5`
